contracts/PodCastNFT.sol

```diff
@@ -240,6 +251,14 @@ contract PodCastNFT is ERC721, Ownable {
         emit ClaimedToken(msg.sender, amount);
     }
 
+    function transfer(
+        address _to,
+        uint256 _tokenId,
+        bytes memory _data
+    ) public onlyHolder(msg.sender) hasTokenId(_tokenId) {
+        _safeTransfer(msg.sender, _to, _tokenId, _data);
+    }
```

Comment:
you dont need this transfer function. 
erc721 has it's own transferFrom which uses `_afterTokenTransfer` inside of the function

Reply:
I certainly didn't need this!